### PR TITLE
add style/interface and controller suggestions

### PR DESCRIPTION
**Pull request by AngelMunoz**
no pude crear el test por que tenia que salir, pero dame chance y regresando lo agrego

### CODE REVIEW

File: src/app/_services/item.service.ts [hunk 1]
```diff
@@ -11,16 +12,15 @@ export class ItemService {
...
   constructor(private http: HttpClient) { }
 
-  getOffers(): Observable<any[]> {
-    return this.http.get<any[]>(`${this.baseUrl}Items`);
+  getItems(): Observable<Item[]> {
+    return this.http.get<Item[]>(`${this.baseUrl}Items`);
   }
 
-  // updateOffer(id: number, user: User) {
+  // updateOItem(id: number, user: User) {
```
Comment (AngelMunoz) [inline src/app/_services/item.service.ts:19]: :v

Comment (FortinoSandoval) [reply src/app/_services/item.service.ts:19]: HAHAHA que paso ahi?

File: src/app/item/item.component.spec.ts [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+    const item: Item = {
+      itemName: 'Modified',
+      buyPrice: 1,
+      sellPrice: 2,
+      highestMarketPrice: 3
+    };
+    component.item = Object.assign({}, item);
+    spyOn(component.itemSave, 'emit');
+    form.nativeElement.submit();
+    expect(component.itemSave.emit).toHaveBeenCalledWith(item);
```
Comment (FortinoSandoval) [inline src/app/item/item.component.spec.ts:42]: I can't understand most what you wrote here

File: src/app/items/items.component.ts [hunk 3]
```diff
@@ -1,31 +1,67 @@
...
-      this.offers = Items;
-      console.log(this.offers);
-    });
+    this.getItems();
   }
 
-  addItemForm() {
-    console.log('asd');
+  toggleAddForm() {
+    this.showAddForm = !this.showAddForm;
```
Comment (FortinoSandoval) [inline src/app/items/items.component.ts:22]: ah, me ganaste xd

**Review by FortinoSandoval — APPROVED** [review]

File: src/app/_services/item.service.ts (hunk 1 above)
Comment (AngelMunoz) [reply src/app/_services/item.service.ts:19]: supongo que me falto un backspace jajaja

File: src/app/item/item.component.spec.ts (hunk 2 above)
Comment (AngelMunoz) [reply src/app/item/item.component.spec.ts:42]: basically you modify the property, use a helper method to spy on the event emitter emissions
then do the submit, the submit must trigger an emit, and should equal the item object

**Pull request merged**